src/basic/process-util.c

```diff
@@ -1083,6 +1078,10 @@ int getenv_for_pid(pid_t pid, const char *field, char **ret) {
                         *ret = value;
                         return 1;
                 }
+
+                sum += r;
```

Comment:
Please don't move this. Having this close to where `r` is assigned makes the code easier to follow.

Reply:
I thought it being closer to sum would make it easier to follow.

Reply:
Please stop.